demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1413,20 +1407,12 @@ export class DemoMeetingApp
       this.priorityBasedDownlinkPolicy.addObserver(this);
     }
 
-    if (!this.enableEventReporting) {
-      this.meetingSession = new DefaultMeetingSession(
-        configuration,
-        this.meetingLogger,
-        deviceController,
-        new NoOpEventReporter()
-      );
-    } else {
-      this.meetingSession = new DefaultMeetingSession(
-        configuration,
-        this.meetingLogger,
-        deviceController
-      );  
-    }
+    this.meetingSession = new DefaultMeetingSession(
+      configuration,
+      this.meetingLogger,
+      deviceController,
+      this.eventReporter
+    );
     this.eventReporter = this.meetingSession.eventReporter;
```

Comment:
Do we need this line anymore?

Reply:
Thanks for the catch, no this is not needed anymore. Removed it.